translations/pl.json

```diff
@@ -1,195 +1,664 @@
+    "selected-dapps": "Wybrane ÐAplikacje",
+    "selected-for-you": "Wybrane dla Ciebie",
+    "send-message": "Wyślij wiadomość",
+    "send-request": "Wyślij prośbę tranzakcji",
+    "send-sending-to": "do",
+    "send-transaction": "Wyślij transakcję",
+    "send-transaction-request": "Wyślij prośbę tranzakcji",
+    "sent-at": "Wyslano w",
+    "set-a-topic": "Utwórz temat",
+    "settings": "Ustawienia",
+    "shake-your-phone": "Znalazłeś błąd lub masz pytanie? Wystarczy ~potrząsnąć~ telefonem!",
+    "share": "Udostępnij",
+    "share-contact-code": "Udostępnij mój kod kontaktowy",
+    "sharing-cancel": "Anuluj",
+    "sharing-copied-to-clipboard": "Skopiowane do schowka",
+    "sharing-copy-to-clipboard": "Skopiuj do schowka",
+    "sharing-share": "Udostępnij...",
+    "show-less": "Pokaż mniej",
+    "show-more": "Pokaż więcej",
+    "show-qr": "Pokaż QR",
+    "sign-in": "Zaloguj się",
+    "sign-in-to-another": "Zaloguj się do innego konta",
+    "sign-in-to-status": "Zaloguj się do Statusu",
+    "sign-later-text": "Sprawdź historię transakcji, aby podpisać tę transakcję",
+    "sign-later-title": "Podpisać transakcję później?",
+    "sign-message": "Podpisz wiadomość",
+    "sign-you-in": "Logujemy Cię…",
+    "signing-message-phrase-description": "Jeśli rozpoznajesz te słowa, wprowadź swoje hasło logowania, aby podpisać transakcję",
+    "signing-phrase-description": "Jeśli rozpoznajesz te słowa, wprowadź swoje hasło logowania, aby podpisać transakcję",
+    "soon": "Wkrótce",
+    "specify-bootnode-address": "Podaj adres bootnode'a",
+    "specify-mailserver-address": "Podaj adres mailserver'a",
+    "specify-name": "Podaj nazwę",
+    "specify-network-id": "Podaj id sieci",
+    "specify-recipient": "Określ odbiorcę…",
+    "specify-rpc-url": "Podaj RPC URL",
+    "start-chat": "Rozpocznij czat",
+    "start-conversation": "Rozpocznij rozmowę",
+    "start-group-chat": "Rozpocznij czat grupowy",
+    "start-new-chat": "Rozpocznij nowy czat",
+    "status": "Status",
+    "status-confirmed": "Potwierdzone",
+    "status-console": "Konsola Status",
     "status-delivered": "Dostarczono",
-    "profile": "Profil",
-    "no-statuses-discovered": "Nie odkryto statusów",
-    "none": "Brak",
-    "removed": "usunięci",
-    "empty-topic": "Pusty temat",
-    "message": "Wiadomość",
-    "here-is-your-passphrase": "Oto twoje specjalne hasło, *zapisz je i przechowuj w bezpiecznym miejscu!* Będzie ci potrzebne podczas procedury odzyskiwania konta.",
-    "image-source-title": "Zdjęcie profilowe ",
-    "left": "pozostało",
+    "status-failed": "Niepowodzenie",
+    "status-not-sent": "Nie wysłano. Dotknij aby zobaczyć opcje",
+    "status-pending": "W oczekiwaniu",
+    "status-prompt": "Utwórz status, aby inni mogli dowiedzieć się, co oferujesz. Możesz również wykorzystać #hashtagi.",
+    "status-seen": "Przeczytane",
+    "status-seen-by-everyone": "Przeczytane przez wszystkich",
+    "status-sending": "Wysyłanie…",
```

Comment:
same here

Reply:
Also like this in Lokalise. The English also has "..."